Demo/Sources/Settings/SettingsView.swift

```diff
@@ -188,6 +189,26 @@ struct SettingsView: View {
         .frame(maxWidth: .infinity)
     }
 
+    @ViewBuilder
+    private func gitHubSection() -> some View {
+#if os(iOS)
+        Section("GitHub") {
+            Button("Project") { GitHub.open(.project) }
+            Button("Source code") { GitHub.open(.apple) }
+                .swipeActions {
+                    Button("Web") { GitHub.open(.web) }
+                        .tint(.yellow)
+                    Button("Documentation") { GitHub.open(.documentation) }
+                        .tint(.red)
+                    Button("Android") { GitHub.open(.android) }
+                        .tint(.green)
+                }
+        }
+#else
+        self
+#endif
```

Comment:
Not sure about the tvOS fallback on `self`. Shouldn't you rather do:

```swift
#if os(iOS)
gitHubSection()
#endif
```

where the GitHub section is inserted in the parent hierarchy?